packages/react/src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,76 @@
+import type { HTMLAttributes } from 'react';
+import React, { forwardRef } from 'react';
+import {
+  InformationSquareFillIcon,
+  CheckmarkCircleFillIcon,
+  XMarkOctagonFillIcon,
+  ExclamationmarkTriangleFillIcon,
+} from '@navikt/aksel-icons';
+import cn from 'classnames';
+
+import { Paragraph } from '..';
+
+import classes from './Alert.module.css';
+
+const icons: Record<Severity, JSX.Element> = {
+  info: (
+    <InformationSquareFillIcon
+      title='Informasjon'
```

Comment:
We should support multiple languages, so I suggest we add a prop for the icon title. We can use "Informasjon" etc. as fallback values.

Reply:
yeah, I was a bit back and forth if we should add this now or not. Thing is to find a good prop name for it. Maybe `iconTitle`, `srIconTitle` or `iconLabel`? I'll add support atleast.

Reply:
I also leaning towards `iconTitle`, short and sweet 💋

Reply:
Added new `iconTitle` prop now @TomasEng @framitdavid